File: swift/Digital_Root.swift
// Digital root of a number
func digitalRoot(of number: Int) -> Int {
  var sum: Int = 0
  let integerList = String(number).map { Int("\($0)")! }
  for i in integerList{
    let integer = Int(i)
    sum += integer
  }
  while String(sum).count > 1{
    sum = digitalRoot(of: sum)
  }
  return sum;
}

File: Python/digital_root.py
def digital_root(num):
    sum = 0
    for digit in str(num):
        sum+=int(digit)
    number = sum
    if int(number)>=10:
        return digital_root(number)
    else:
        return number
print("Digital root of 932 = "+str(digital_root(932)))
